src/ResponsePrototype.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * This file is part of the prooph/event-store-http-middleware.
+ * (c) 2018-2018 prooph software GmbH <contact@prooph.de>
+ * (c) 2018-2018 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStore\Http\Middleware;
+
+use Psr\Http\Message\ResponseInterface;
+
+interface ResponsePrototype extends ResponseInterface
```

Comment:
Maybe it's better to rely on a string instead of an interface that's never implemented. What do you think?